R/data.R

```diff
@@ -15,6 +15,8 @@
 #' data fields.
 #' @param chromosomeField A character. Specify the name of chromosome
 #' data fields.
+#' @param genomicFieldsToConvert Define the genomic fields from the data
+#' in list format.
```

Comment:
I would prefer to use the same argument definition as the [gosling js docs](http://gosling-lang.org/docs/data) highlighting that it's experimental.
```
Experimental Proerty. Each object follows the format {"chromosomeField":"string","genomicFields":"string[]"} ( )
```